src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -210,12 +238,30 @@ private static void init(){}
         public static final TagKey<Item> COBBLESTONE_INFESTED = tag("cobblestone/infested");
         public static final TagKey<Item> COBBLESTONE_MOSSY = tag("cobblestone/mossy");
         public static final TagKey<Item> COBBLESTONE_DEEPSLATE = tag("cobblestone/deepslate");
+        public static final TagKey<Item> CLUSTERS = tag("clusters");
         public static final TagKey<Item> CROPS = tag("crops");
         public static final TagKey<Item> CROPS_BEETROOT = tag("crops/beetroot");
         public static final TagKey<Item> CROPS_CARROT = tag("crops/carrot");
         public static final TagKey<Item> CROPS_NETHER_WART = tag("crops/nether_wart");
         public static final TagKey<Item> CROPS_POTATO = tag("crops/potato");
         public static final TagKey<Item> CROPS_WHEAT = tag("crops/wheat");
+        public static final TagKey<Item> DYED_ITEMS = tag("dyed_items");
```

Comment:
Same as the blocks, I see no need to mention the registry name. `dyed` is sufficient, and provides parity with the block tag name. The name right now is confusing as it might suggest that glass *blocks* do not belong in the tag, when I assume its meant to hold any items that might be considered dyed.

Reply:
Hmm. that is a good argument. I'll make the change and we will see if people are cool with the base tag name being just `dyed.json`